test/starlark_tests/targets_under_test/ios/BUILD

```diff
@@ -2396,3 +2396,100 @@ objc_library(
     srcs = ["//test/starlark_tests/resources:main.m"],
     runtime_deps = [":fmwk_8_0_minimum"],
 )
+
+# ---------------------------------------------------------------------------------------
+
```

Comment:
Can we add a line that says what these targets are for? The one above should have one too, I'm not sure why it wasn't added.